docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+
+While there, you might also want to choose a **Category** for your article (or add your own) from the list.
+
+#### Turn your draft into a publication
+
+By default, your new article would be saved as a draft. Let's not be too shy and publish it right away:
+
+1. From the **Status** dropdown at the bottom of the page, choose **published**.
+2. Click on **Save** at the top of the window.
+
+![Animated GIF to create an article](../assets/quick-start-guide/qsg-starters-part2-03-write_publish_article.gif)
+
+You have just created and published your first article, `Hello World!`. You can find it in the [_Collection types > Articles_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::article.article?page=1&pageSize=10&_sort=id:DESC) view.
+
+### Step 3: Update the `Homepage` single type
+
+It's time to make this blog a bit more yours.
+
+Click on [_Single Types > Homepage_](http://localhost:1337/admin/plugins/content-manager/singleType/application::homepage.homepage) in the left-hand menu. Let's edit this homepage:
+
+1. Hover the picture in the **ShareImage field** and click the **Edit** icon <Fa-PencilAlt />.
+2. Click on **Replace Media** and upload an image from your computer. This image will represent your blog when sharing an article on social medias.
+3. Click **Finish** twice.
+4. At the bottom of the page, update the **Title** to `My Wonderful Strapi Blog` in the **Hero** field group.
+5. Finally, click **Save**.
+
+### Step 4: Restart the servers to reflect latest changes
+
+Gatsby is a static-site generator. It means that you need to restart the servers for changes to appear on the frontend:
+
+1. In your terminal, press `Ctrl-C` to stop the servers.
+2. Make sure you are in the `my-project` folder. If not, type `cd my-project` and press `Enter`.
+3. Restart the servers by typing `yarn develop` (or `npm run develop`) and press `Enter`.
+
+After a few moments, you should see your blog with its updated title running at [http://localhost:8000](http://localhost:8000). The `Hello World!` article you have just created is also visible at the bottom of the page.
+
+![GIF: Updated Gatsby blog frontend](../assets/quick-start-guide/qsg-starters-part2-04-restart_servers.gif)
+
+:::warning CONGRATULATIONS! 🥳
+Now you know how to use Strapi to create and update your blog. Keep on creating amazing content!
+:::
+
+## ⏩  What to do next?
+
+The beauty of using Strapi [starters](https://strapi.io/starters) is that the Strapi backend comes with a frontend ready out-of-the-box. Now you probably want to show your shiny new website to the world! The next step is then to deploy both the Strapi backend and the frontend on the platforms of your choice:
+
+👉 For the Strapi backend, we offer various deployment options — see our [Deployment guides](/developer-docs/latest/setup-deployment-guides/deployment.md).
```

Comment:
In the User Guide we use this format when we mention/refer a page.

```diff
- 👉 For the Strapi backend, we offer various deployment options — see our [Deployment guides](/developer-docs/latest/setup-deployment-guides/deployment.md).
+ 👉 For the Strapi backend, we offer various deployment options (see [Deployment guides](/developer-docs/latest/setup-deployment-guides/deployment.md)).
```

Reply:
Indeed, it's in our guidelines. But here I really wanted to emphasize the call to action. Any thoughts, @meganelacheny ?

Reply:
I would use the same format as for the user guide, and perhaps add a bit more explanations ("various deployment options" is quite vague) instead of just emphasising a link. This sentence feels a bit light especially compared to the one right after.